<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>noteForm</class>
 <widget class="QWidget" name="noteForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="titleLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="footerWidget" native="true">
     <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
      <item>
       <widget class="QPushButton" name="savePushButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePushButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
